src/app/dataset-view/additional-components/dataset-settings-component/dataset-settings.model.ts

```diff
@@ -0,0 +1,29 @@
+export enum SettingsTabsEnum {
+    GENERAL = "general",
+    SCHEDULING = "scheduling",
+}
+
+export enum PollingGroupEnum {
+    TIME_DELTA = "TimeDelta",
+    CRON_EXPRESSION = "CronExpression",
+}
+
+export enum ThrottlingGroupEnum {
+    AWAIT_FOR = "awaitFor",
+    AWAIT_UNTIL = "awaitUntil",
+}
+
+export interface SchedulingSettings {
+    updatesState: boolean;
+    pollingGroup: {
+        pollingSource: PollingGroupEnum;
+        timeDelta?: number;
+        timeSegment?: string;
```

Comment:
This should be a separate structure, they always come together.

Reply:
Removed, this interface is not used